akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -0,0 +1,90 @@
+# Rolling Updates
+
+A rolling update is the process of replacing one version of the system with another without downtime.
+The changes can be new code, changed dependencies such as new Akka version, or modified configuration.
+
+In Akka, rolling updates are typically used for a stateful Akka Cluster where you can't run two separate clusters in
+parallel during the update, for example in blue green deployments.
+
+For rolling updates related to Akka dependency version upgrades and the migration guides, please see
+@ref:[Rolling Updates and Akka versions](../project/rolling-update.md)
+
+#### This document covers: 
```

Comment:
Do we need an extra TOC here?

Reply:
No but my thinking was if it were me, I'd find it useful to see if the doc had what I was looking for vs having to scan it, for anyone looking for their specific issues.